doc/guides/device_mgmt/smp_groups/smp_group_0.rst

```diff
@@ -0,0 +1,259 @@
+    +========+==============+================+
+    | ``1``  | ``0``        |  ``2``         |
+    +--------+--------------+----------------+
+
+CBOR data of response command:
+
+.. code-block:: none
+
+    {
+        (str)"tasks" : {
+            (str)<task_name> : {
+                (str)"prio"         : (uint)
+                (str)"tid"          : (uint)
+                (str)"state"        : (uint)
+                (str)"stkuse"       : (uint)
+                (str)"stksiz"       : (uint)
+                (str)"cswcnt"       : (uint)
+                (str)"runtime"      : (uint)
+                (str)"last_checkin" : (uint)
+                (str)"next_checkin" : (uint)
+            }
+            ...
+        }
+        (str)"rc" : (int)
+    }
+
+
+where
+
+.. table::
+    :align: center
+
+    +-----------------------+---------------------------------------------------+
+    | <task_name>           | string identifying task                           |
+    +-----------------------+---------------------------------------------------+
+    | "prio"                | task priority                                     |
+    +-----------------------+---------------------------------------------------+
+    | "tid"                 | numeric task ID                                   |
+    +-----------------------+---------------------------------------------------+
+    | "state"               | numeric task state                                |
+    +-----------------------+---------------------------------------------------+
+    | "stkuse"              | task's/thread's stack usage                       |
+    +-----------------------+---------------------------------------------------+
+    | "stksiz"              | task's/thread's stack size                        |
+    +-----------------------+---------------------------------------------------+
+    | "cswcnt"              | task's/thread's context switches                  |
+    +-----------------------+---------------------------------------------------+
+    | "runtime"             | task's/thread's runtime in "ticks"                |
+    +-----------------------+---------------------------------------------------+
+    | "last_checkin"        | set to 0 by Zephyr                                |
+    +-----------------------+---------------------------------------------------+
+    | "next_checkin"        | set to 0 by Zephyr                                |
+    +-----------------------+---------------------------------------------------+
+    | "rc"                  | :ref:`mcumgr_smp_protocol_status_codes`           |
+    +-----------------------+---------------------------------------------------+
+
+.. note::
+    The unit for "stkuse" and "stksiz" is system dependent and in case of Zephyr
+    this is number of 4 byte words.
+
+MPSTAT command
+==============
+
+The command lacks formal definition except for being assigned ID. It is not really
```

Comment:
Here's the in-code documentation: https://github.com/NordicSemiconductor/Android-nRF-Connect-Device-Manager/blob/7b9958618dce691ce6abb568ff0ed3bffc821d8f/mcumgr-core/src/main/java/io/runtime/mcumgr/response/dflt/McuMgrMpStatResponse.java#L18
This is a READ-ONLY command, so only OP = 0.

Reply:
Thanks for info, done.